CMakeLists.txt

```diff
@@ -127,8 +127,12 @@ cmake_dependent_option(
 
 set_local_or_system_option(
     "GTEST" OFF
-    "Build the GoogleTest framework by providing your own copy of the repo in ext/googletest (see Local Builds in README)"
+    "Use the GoogleTest framework. Enables avif C++ tests that depend on GoogleTest. Has no effect unless AVIF_BUILD_TESTS is ON."
 )
+if(NOT AVIF_GTEST STREQUAL "OFF")
+    set(AVIF_ENABLE_GTEST ON)
```

Comment:
no need to define an extra variable anymore: just replace the two instances of it below with AVIF_GTEST. Same in tests/CMakeLists.txt

Reply:
I though it was easier like this (and fewer changes) than checking `NOT AVIF_GTEST STREQUAL "OFF"` everywhere, no? Or does `if(AVIF_GTEST)` work? (I didn't think so since it's a string). I'm happy to change it though if you think it's better.

Reply:
Hmm, I thought it would work with `"OFF"`, https://cmake.org/cmake/help/latest/command/if.html#constant but maybe you can try and see if it works. If not, if it is equal to `"OFF"`, just set it to `OFF`

Reply:
That does seem to work. TIL!